src/kernels/MIOpenBatchNormActivFwdTrainSpatial.cl

```diff
@@ -109,21 +109,21 @@ MIOpenBatchNormActivFwdTrainSpatial(float INHW,
 
     if(lid < MIO_BN_SEGMENT)
     {
-#if MIOPEN_USE_FP16 == 0
+#if (MIOPEN_USE_FP16 == 0) || (MIOPEN_USE_BF16 == 1)
```

Comment:
This define isn't for mixed, and we shouldn't do this for BF16 since it might cause problems.

Reply:
Oh it's fine actually since the define doesn't exit, but ill remove it.